bedrock/base/templates/includes/protocol/footer/footer-refresh.html

```diff
@@ -0,0 +1,129 @@
+{#
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ #}
+
+ <footer class="moz24-footer {% if theme_class %}{{ theme_class }}{% endif %}" id="colophon">
```

Comment:
**Question:** is `theme_class` needed still? If not, can we remove it?

Reply:
I don't think theme is supported for MVP, I will remove this for now